streaming/java/streaming-api/src/main/java/org/ray/streaming/api/context/RuntimeContext.java

```diff
@@ -1,5 +1,14 @@
 package org.ray.streaming.api.context;
 
+import java.util.Map;
+import org.ray.streaming.state.backend.KeyStateBackend;
+import org.ray.streaming.state.keystate.desc.ListStateDescriptor;
+import org.ray.streaming.state.keystate.desc.MapStateDescriptor;
+import org.ray.streaming.state.keystate.desc.ValueStateDescriptor;
+import org.ray.streaming.state.keystate.state.ListState;
+import org.ray.streaming.state.keystate.state.MapState;
+import org.ray.streaming.state.keystate.state.ValueState;
+
 /**
  * Encapsulate the runtime information of a streaming task.
  */
```

Comment:
Maybe we should consider using StreamRuntimeContext, since we are building fusion computing in Ray with multiple computer engines. Each of them might have its own RuntimeContext.

Reply:
Good suggestion, I agree `StreamRuntimeContext` is better.